Externals.cfg

```diff
@@ -15,37 +15,37 @@ local_path = components/cice
 required = True
 
 [cime]
-tag = cime5.6.32
+branch = add2300extsspanom
```

Comment:
No - let's use the ESMCI repo

Reply:
Yes, once there is a cime tag I can update to it.

Reply:
* [new tag]             cime5.6.33 -> cime5.6.33

Reply:
Awesome. OK, testing the CLM branch with that tag now. Will update this PR with the cime and CLM tag once that's done.

Reply:
OK, I've updated to the tags.